tests/neg/i22906.scala

```diff
@@ -0,0 +1,5 @@
+//> using scala 3.7.0-RC1
+//> using options -rewrite -indent
```

Comment:
This is nominally a no-op without `-source 3.0-migration` but would throw; it requires the other options to demonstrate the behavior.

Reply:
updated to document my comment that vulpix doesn't demonstrate the error.